test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +64,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
```

Comment:
Since this is a view for `Area1` which used to be `Aria`, should restore fact this was linking to root action.